client/src/components/Home/LandingPage/LandingPage.css

```diff
@@ -7,7 +7,7 @@
       height: 95vh;
```

Comment:
makes this 100vh . . .  This will fix it for all screen size

Reply:
Okay, but when I tested I didn't got that border issue again with any screen size

Reply:
See this on iPad
![Screenshot from 2020-10-01 19-53-11](https://user-images.githubusercontent.com/48787278/94822040-de007000-041f-11eb-8476-9b45e4a42ded.png)

Reply:
Made a commit.